apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -0,0 +1,18 @@
+import { IMultilingualText } from '@coscrad/api-interfaces';
+
+export interface MultilingualTextPresenterProps {
+    text: IMultilingualText;
+}
+
+export const MultilingualTextPresenter = ({
+    text: { items },
+}: MultilingualTextPresenterProps): JSX.Element => (
+    <>
+        {/* TODO We need to separate the original in some way */}
+        {items.map(({ languageCode, text, role }) => (
```

Comment:
Would we always display the original?

Reply:
What I'm thinking is that we show the original but with some kind of way to hover or click to see the others. Down the road, we would support localization and show the one in the "selected language" (coscrad.org/resources/songs?lang=hai)
or fall back to the original (indicating via colour or presentation that we've done so) if there is no translation in the selected language,

Reply:
But for now, we just show all variants.